src/components/pager/pager.stories.mdx

```diff
@@ -0,0 +1,250 @@
+import { Meta, Props, Preview, Story } from "@storybook/addon-docs/blocks";
+import Pager from ".";
+import { useState } from "react";
+import {
+  text,
+  boolean,
+  select,
+  withKnobs,
+  object
+} from "@storybook/addon-knobs";
+import { action } from "@storybook/addon-actions";
+import useMediaQuery from "../../hooks/useMediaQuery/index.js"
+
+<Meta title="Design System/Pager" />
+
+# Pager
+Pagination Component.
+
+### Default usage:
+
+<Preview>
+  <Story name="default" parameters={{ info: { disable: true }}} >
+    {() => {
+      const totalRecords = text("totalRecords", "100");
+      const showPageSizeSelection = boolean(
+        "showPageSizeSelection",
+        false
+      );
+      const pageSize = select("pageSize", {
+        one: 1,
+        10: 10,
+        25: 25,
+        50: 50,
+        100: 100
+      }, 10);
+      const currentPage = text("currentPage", "1");
+      const pageSizeSelectionOptions = object("pageSizeSelectionOptions",
+        [
+          { id: "1", name: 1 },
+          { id: "10", name: 10 },
+          { id: "25", name: 25 },
+          { id: "50", name: 50 },
+          { id: "100", name: 100 }
+        ]
+      );
+      return (
+        <Pager
+          totalRecords={ totalRecords }
+          showPageSizeSelection={ showPageSizeSelection }
+          pageSize={ pageSize }
+          currentPage={ currentPage }
+          onPagination={ () => {} }
+          pageSizeSelectionOptions={ pageSizeSelectionOptions }
+        />
+      );
+    }}
+  </Story>
+</Preview>
+
+### With disabled Page Size Selection:
+
+<Preview>
+  <Story name="disabled page size" parameters={{ info: { disable: true }}} >
+    <Pager
+      totalRecords={ 100 }
+      onPagination={ () => {} }
+    />
+  </Story>
+</Preview>
+
+### Hiding Pager elements:
+Unlike with hiding the page size select component, the remaining elements that make up the `Pager` are rendered by 
+default: in order for them to be hidden you must pass `false` to the corresponding `show...` prop (see prop table below). 
+In the example below the `First` and `Last` buttons and `totalRecords` label have been hidden by setting the 
+`showFirstButton`, `showLastButton` and `showTotalRecords` props to false.
+
+<Preview>
+  <Story name="hiding pager elements" parameters={{ info: { disable: true }}} >
```

Comment:
Minor note that you can change if you want - you can just put this `parameters={{ info: { disable: true } }}` on the `Meta` tag at the top rather than on every story